app/services/weets.js

```diff
@@ -1,15 +1,30 @@
 const axios = require('axios');
+const { Weet } = require('../models');
+const logger = require('../logger');
 
 const { randomWeetEndpoint } = require('../../config').common.weetApi;
-const { externalApiError, notFoundError } = require('../errors');
+const { externalApiError, notFoundError, databaseError } = require('../errors');
+const { databaseErrorMessage } = require('../helpers/constants');
 
-exports.getWeet = () =>
-  axios
-    .get(randomWeetEndpoint)
-    .then(response => response.data.data[0])
-    .catch(error => {
-      if (error.response && error.response.status === 404) {
-        return notFoundError(error.message);
-      }
-      return externalApiError(error.message);
-    });
+exports.getWeet = async () => {
+  try {
+    const response = await axios.get(randomWeetEndpoint);
+    return response.data.data[0];
+  } catch (error) {
+    logger.error(error.message);
+    if (error.response && error.response.status === 404) {
```

Comment:
Use the constant for this 404

Reply:
I changed the error messages with the respective constants for notFoundError and externalApiError

Reply:
I mean the 404 number 😛

Reply:
ok